src/lib/Scenes/Show/Components/ShowHeader/Components/Carousel.tsx

```diff
@@ -0,0 +1,64 @@
+import { Box, Flex, space } from "@artsy/palette"
+import OpaqueImageView from "lib/Components/OpaqueImageView"
+import React from "react"
+import { Dimensions, ScrollView } from "react-native"
+import styled from "styled-components/native"
+
+const ITEM_HEIGHT = 350
+
+const { width: windowWidth } = Dimensions.get("window")
+
+const ImageView = styled(OpaqueImageView)<{ isFirst: boolean; aspectRatio: number }>`
+  height: ${ITEM_HEIGHT};
+  margin-top: ${p => (p.isFirst ? 150 : 0)};
+  ${p =>
+    p.isFirst &&
+    `
+      height: 200;
```

Comment:
May end up changing the logic around computing the scaled down first image but I tested this on a few shows and it works fine for now.